cni/network/network_windows.go

```diff
@@ -378,8 +378,8 @@ func determineWinVer() {
 	}
 }
 
-func getNATInfo(executionMode string, ncPrimaryIPIface interface{}, multitenancy, enableSnatForDNS bool) (natInfo []policy.NATInfo) {
-	if executionMode == string(util.V4Swift) {
+func getNATInfo(executionMode, ipamMode string, ncPrimaryIPIface interface{}, multitenancy, enableSnatForDNS bool) (natInfo []policy.NATInfo) {
+	if executionMode == string(util.V4Swift) && ipamMode != string(util.V4Overlay) {
```

Comment:
can you add UT for this scenario for overlay vs podsubnet?

Reply:
added